subsys/net/lib/nrf_provisioning/include/nrf_provisioning_http.h

```diff
@@ -65,9 +65,8 @@ struct nrf_provisioning_http_context {
 	 */
 	int32_t timeout_ms;
 	/** Authentication string: Bearer Token
-	 * If no JWT is provided, and CONFIG_NRF_PROVISIONING_AUTOGEN_JWT
 	 * is enabled, then one will be generated automatically with
-	 * CONFIG_NRF_PROVISIONING_AUTOGEN_JWT_VALID_TIME_S as its lifetime in
+	 * CONFIG_NRF_PROVISIONING_JWT_MAX_VALID_TIME_S as its lifetime in
 	 * seconds.
 	 * If Attestation Token based authentication is enabled leaving this empty is mandatory,
 	 * unless a static token is used for testing purposes.
```

Comment:
Please remove this whole `char *nrf_provisioning_http_context.auth` as it is not used anywhere.

Looks like it should have been removed in https://github.com/nrfconnect/sdk-nrf/pull/12971 but only the Kconfigs were removed and the use of `char *auth`.
So this is just leftover.